openbb_terminal/miscellaneous/library/trail_map_optimization.csv

```diff
@@ -13,7 +12,6 @@ portfolio.po.maxret,openbb_terminal.portfolio.portfolio_optimization.po_model.ge
 portfolio.po.maxsharpe,openbb_terminal.portfolio.portfolio_optimization.po_model.get_maxsharpe,
 portfolio.po.maxutil,openbb_terminal.portfolio.portfolio_optimization.po_model.get_maxutil,
 portfolio.po.minrisk,openbb_terminal.portfolio.portfolio_optimization.po_model.get_minrisk,
-portfolio.po.mktcap,openbb_terminal.portfolio.portfolio_optimization.po_model.get_mktcap,
```

Comment:
dont we still have this one

Reply:
nope?
![image](https://user-images.githubusercontent.com/61340027/218164084-72e65439-2659-47f2-a895-2bc994fafc5d.png)

Reply:
rip